### PR TITLE
create header component

### PR DESCRIPTION
**Pull request by lalidiaz**
## Goal

This pull request is regarding task:

https://github.com/Pocket/curation-tools-frontend/issues/61

## Todos:
- [x] Create header component.
 
Thanks. 

Laura.

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React from "react";
```
Comment (kkyeboah) [inline src/components/Header/index.tsx]: Why did you choose to name the file `index.tsx` instead of `Header.tsx`?

**Review by kkyeboah — DISMISSED** [review]
This is really great work 👍 We just have a question about your file naming choice.

Please revert the changes to `App.tsx` and we will merge this!

File: src/components/Header/index.tsx (hunk 1 above)
Comment (lalidiaz) [reply src/components/Header/index.tsx]: I used index.tsx just not to be redundant in your import statement. But I am happy to change it as Header.tsx. Let me know if you want me to change it to index.tsx as before please :)

Comment (kkyeboah) [reply src/components/Header/index.tsx]: No, you don't have to change it. We were just curious as to why? 

Nice work!

**Comment by lalidiaz** [comment]
@jpetto @kkyeboah  
This pull request is ready to merge to main! :)

File: src/components/Header/index.tsx (hunk 1 above)
Comment (lalidiaz) [reply src/components/Header/index.tsx]: Thank you! 👍🏽

**Review by kkyeboah — COMMENTED** [review]
Could you revert `App.tsx` to what it was before you added your component?

For reference: You can simply run this command `git checkout main src/App.tsx`.
This should check out this file: https://github.com/Pocket/curation-tools-frontend/blob/main/src/App.tsx from the main branch

**Comment by lalidiaz** [comment]
> Could you revert `App.tsx` to what it was before you added your component?
> 
> For reference: You can simply run this command `git checkout main src/App.tsx`.
> This should check out this file: https://github.com/Pocket/curation-tools-frontend/blob/main/src/App.tsx from the main branch

No problem :). Here are the changes you requested.

File: src/App.tsx [hunk 2]
```diff
@@ -2,12 +2,11 @@ import React from 'react';
...
+
 function App() {
   return (
     <div className="App">
       <!-- your component goes here to test the rendering! -->
     </div>
   );
-}
-
-export default App;
```
Comment (kkyeboah) [inline src/App.tsx]: please don't delete this line.

**Review by kkyeboah — COMMENTED** [review]
Could you please make sure there are not difference between `App.tsx` file in your PR and the `main` branch

File: src/App.tsx (hunk 2 above)
Comment (lalidiaz) [reply src/App.tsx]: Sorry for that, done.

**Review by kkyeboah — APPROVED** [review]

**Pull request merged**